docs/source/conf.py

```diff
@@ -241,9 +241,9 @@ def copy_documentation_examples(source_folder, target_folder):
         for line in content.split("\n"):
             line = line.strip(" \t")
             if line.startswith("examples/"):
-                notebooks_to_include.append(line.split("/", 1)[1])
+                files_to_include.append(line.split("/", 1)[1])
 
-    for notebook in notebooks_to_include:
+    for notebook in files_to_include:
```

Comment:
Perhaps just rename to `for file in files_to_include`, as they are not strictly just notebooks anymore.

Reply:
done